static/src/javascripts/es6/projects/common/modules/crosswords/clues.js

```diff
@@ -1,9 +1,10 @@
 /* jshint newcap: false */
+import classNames from 'classnames';
+import React from 'react';
+
 import _ from 'common/utils/_';
-import React from 'react/addons';
 
-var classSet = React.addons.classSet,
-    Clue = React.createClass({
+var Clue = React.createClass({
```

Comment:
In the latest version of `React` you can use ES6 classes, too. See https://github.com/guardian/quizzes/blob/master/lib/components.jsx for example.